ballerina/metadata_types.bal

```diff
@@ -31,18 +31,33 @@ public type SQLMetadata record {|
     map<JoinMetadata> joinMetadata?;
 |};
 
+
 public type TableMetadata record {|
     string[] keyFields;
     function (string[]) returns stream<record {}, Error?> query;
     function (anydata) returns record {}|InvalidKeyError queryOne;
     map<function (record {}, string[]) returns record {}[]> associationsMethods = {};
 |};
 
+public type SheetMetadata record {|
+    string entityName;
+    string tableName;
+    map<SheetFieldMetadata> fieldMetadata;
+    string[] keyFields;
+    string range;
+    map<string> dataTypes = {};
```

Comment:
```suggestion
    map<typdesc> dataTypes = {};
```

Can consider making this a map of `typedesc`

Reply:
Will create a issue for this change as persist tools are also dependant on this